src/d3a/models/area/__init__.py

```diff
@@ -43,14 +43,13 @@
 log = getLogger(__name__)
 
 
+# TODO: As this is only used in the unittests, please move to the tests package
```

Comment:
I would say to delete this and replace the usages of this class with constants_limits.GlobalConfig class. If you can change the TODO to reflect that it would be great

Reply:
Ok, will do!